pkgs/misc/themes/arc/default.nix

```diff
@@ -1,43 +1,55 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gnome3, gtk-engine-murrine }:
+{ stdenv, fetchFromGitHub, sassc, autoreconfHook, pkgconfig, gnome3
+, gnome-themes-standard, gtk-engine-murrine, optipng, inkscape}:
 
 let
-  # treat versions newer than 3.22 as 3.22
-  gnomeVersion = if stdenv.lib.versionOlder "3.22" gnome3.version then "3.22" else gnome3.version;
+  # per README: For GNOME 3.24 and 3.26, use --with-gnome=3.22
+  gnomeVersion = if gnome3.version == "3.24" || gnome3.version == "3.26" then "3.22" else gnome3.version;
```

Comment:
Maybe it is better to rely on the automatic detection of the gnome version provided by the package itself. Reading its code reveals it is in fact based on the gtk+ version, obtained with the help of pkg-config (`pkg-config --modversion gtk+-3.0`). It seems that in future release this will be changed.

So I suggest to remove those explicit handling of the gnome version.